### PR TITLE
Increase timeouts for tests 5990 and 5989

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
increase timeout for updating ssp cr for test ids 5990 and 5989
**Release note**:
```
NONE

```

### CODE REVIEW

**Comment by akrejcir** [comment]
This is not needed. `waitUntilDeployed()` is already called in `BeforeEach()` in the outer scope.

File: tests/webhook_test.go [hunk 1]
```diff
@@ -164,7 +164,7 @@ var _ = Describe("Validation webhook", func() {
 					foundSsp = getSsp()
 					foundSsp.Spec.TemplateValidator.Placement = &placementAPIValidationValidPlacement
 					return apiClient.Update(ctx, foundSsp, client.DryRunAll)
-				}, time.Second, tenSecondTimeout).ShouldNot(HaveOccurred(), "failed to update SSP CR with valid template-validator placement fields")
+				}, time.Second, twentySecondTimeout).ShouldNot(HaveOccurred(), "failed to update SSP CR with valid template-validator placement fields")
```
Comment (akrejcir) [inline tests/webhook_test.go:167]: The first optional parameter of `Eventually` is timeout and the second is polling interval.

**Review by akrejcir — CHANGES_REQUESTED** [review]

File: tests/webhook_test.go (hunk 1 above)
Comment (ksimon1) [reply tests/webhook_test.go]: good catch :D

**Comment by ksimon1** [comment]
/override ci/prow/e2e-upgrade-functests 
/override ci/prow/e2e-functests
I am overriding tests, as the errors are not related to this PR
/approve

**Pull request merged**